bot/telegram_bot.py

```
@@ -656,12 +656,37 @@ async def _cmd_run_swarm_task(self, update: Update, ctx: ContextTypes.DEFAULT_TY
             await update.message.reply_text("Usage: /run\\_swarm\\_task <your task description>")
             return
         await update.message.reply_text(f"🐝 Swarm executing: _{request[:60]}_…", parse_mode="Markdown")
+
+        # Prefer coordinator dispatch first (easy to mock in tests and consistent with command routing).
+        dispatched: Dict[str, Any] = {}
+        try:
+            dispatched = await self._dispatch_task("swarm_task", {"task": request}, update)
+        except Exception as exc:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)